src/App.css

```diff
@@ -1,38 +1,24 @@
-.App {
-  text-align: center;
-}
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+:root {
+  --font-family: 'Inter', sans-serif;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+html {
+  height: 100%;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+body {
+  height: 100%;
+  margin: 0;
+  font-size: 1.125rem;
+  font-family: var(--font-family);
 }
 
-.App-link {
-  color: #61dafb;
+input {
+  font-family: var(--font-family);
```

Comment:
You set this in the body, I don't think it also needs to be set here..?

Reply:
Apparently `input` and `button` elements don't inherit the font from the `body` element by default, so I set `font-family: inherit` for both of these